daphne/src/audit_log.rs

```diff
@@ -0,0 +1,42 @@
+use crate::{messages::TaskId, DapTaskConfig};
+
+pub trait AuditLog {
+    fn on_aggregation_job_init_req(
```

Comment:
Why do we need a separate callback for agg init and and continue? How about a single callback for aggregate requests?

Reply:
we don't, I can update it to take a boolean if that is preferred.

Reply:
I'd rather keep boolean parameters to a minimum, they make code hard to read. You can instead define a simple two variant enum, same performance, much better readability
```rs
enum AuditAction {
    Init,
    Continue,
}
```